tests/FSharpPlus.Tests/General.fs

```diff
@@ -500,6 +501,33 @@ module Foldable =
         Assert.IsInstanceOf<Option<list<int>>> (Some sortedList)
         Assert.IsInstanceOf<Option<seq<int>>> (Some sortedSeq)
 
+    [<Test>]
+    let intersperse() =
+        Assert.AreEqual ("a,b,c,d,e", intersperse ',' "abcde")
+        Assert.AreEqual (["a";",";"b";",";"c";",";"d";",";"e"], intersperse "," ["a";"b";"c";"d";"e"])
+
+    [<Test>]
+    let readOnlyIntercalate() =
+        Assert.AreEqual ("Lorem, ipsum, dolor", intercalate ", " ["Lorem"; "ipsum"; "dolor"])
+        Assert.AreEqual ("Lorem, ipsum, dolor", intercalate ", " (ReadOnlyCollection( ["Lorem"; "ipsum"; "dolor"] |> List.toArray )))
```

Comment:
Any good reason why instead of:

`["Lorem"; "ipsum"; "dolor"] |> List.toArray`

use array brackets directly `[|` `|]` and save some keystrokes?

`[|"Lorem"; "ipsum"; "dolor"|]`

Reply:
No good reason at all. 😅